layouts/shortcodes/observability_pipelines/configure_existing_pipelines/source_env_vars/amazon_s3.md

```diff
@@ -0,0 +1,9 @@
+- Amazon S3 SQS URL
+    - The URL of the SQS queue to which the S3 bucket sends the notification events.
+    - Stored as the environment variable: `DD_OP_SOURCE_AWS_S3_SQS_URL`
```

Comment:
I won't keep flagging these because I don't want to spam you, but grammatically you don't need the colon after "environment variable," and you don't need a period at the end of bullets that are not complete sentences. But both are minor so as long as whatever you're doing is consistent, I think it's fine!

Reply:
Thanks for pointing this out! I'll open a separate PR later to fix it for all instances.